lib/src/code_generator/objc_interface.dart

```diff
@@ -106,6 +106,22 @@ class _ObjCBuiltInFunctions {
       func.addDependencies(dependencies);
     }
   }
+
+  void generateNSStringUtils(Writer w, StringBuffer s) {
+    // Generate a constructor that wraps stringWithCString.
+    s.write('  factory NSString(${w.className} _lib, String str) {\n');
```

Comment:
Why not add an extension on `String` with a method `toNSString()` instead?

Reply:
Done. I think it makes sense to keep the constructor though, because this the main way of constructing an NSString, so it should have a constructor.